maintainers.html

```diff
@@ -0,0 +1,70 @@
+---
+layout: simple-page
+title: RIOT Maintainers
+subtitle: RIOT is nurtured by a strong community of maintainers.
+---
+<main>
+  <!-- ======= Maintainers Section ======= -->
+  <section id="maintainers">
+    <div class="container">
+      <p>
+      This is the current list of maintainers within the RIOT community with there area of
+      expertise within the
+      <a href="https://github.com/RIOT-OS/RIOT/" target="_blank">RIOT codebase</a>.
+
+      If a maintainer is marked with <q>Has no chosen area of expertise</q>, they did not add any
+      ownership within
+      <a href="https://github.com/RIOT-OS/RIOT/blob/master/CODEOWNERS" target="_blank">
+        <tt>CODEOWNERS</tt>
+      </a>. This does not mean that they do not feel responsible for any part
+      of the code base, they just did not declare it.
+      </p>
+
+      <p id="maintainer-list">
+        <div class="row row-cols-1 g-4">
+        {% assign maintainer_by_login = site.data.riot_maintainers | sort_natural: "login" %}
+        {% for maintainer in maintainer_by_login %}
+          <div class="col">
+            <div class="card shadow rounded">
+              <div class="card-body">
+                <h5 class="card-title">
+                  <a href="{{ maintainer.html_url }}" target="_blank"><img class="maintainers-img" src="{{ maintainer.avatar_url }}" alt="{{ maintainer.login }} avatar" /></a>
+                  <a href="{{ maintainer.html_url }}" target="_blank">
+                    @{{ maintainer.login }}
+                    {% if maintainer.name or maintainer.name != maintainer.login %}<small>({{ maintainer.name }})</small>{% endif %}
+                  </a>
+                </h5>
+                <ul>
+                {% if maintainer.owner %}
+                  <li><strong>Is one of the GitHub owners of RIOT</strong></li>
+                {% endif %}
+                {% if maintainer.admin %}
+                  <li><strong>Is one of the GitHub admins of RIOT</strong></li>
+                {% endif %}
+                {% for area in maintainer.areas %}
+                  <li><tt>{{ area }}</tt></li>
+                {% else %}
+                  <li>Has no chosen area of expertise</li>
+                {% endfor %}</ul>
+              </div>
+            </div>
+          </div>
+          {% endfor %}
+        </div>
+      </p>
+      <p>
+      This list is generated by combining the information from the maintainers, owners and
+      admin teams from the RIOT-OS GitHub organization and the
+      <a href="https://github.com/RIOT-OS/RIOT/blob/master/CODEOWNERS" target="_blank">
+        <tt>CODEOWNERS</tt>
+      </a> file within the RIOT repository.
+      </p>
+      
+      <p>
+      If you are a maintainer and want to declare ownership for a part of a code base (and
+      receive notifications on pull requests against it), please add yourself and the path to
```

Comment:
please add yourself and the path to the part of the code base you want to be responsible for to CODEOWNERS, and issue a pull request.